lib/api/core/auth/formatProcessing.js

```diff
@@ -31,12 +31,14 @@ module.exports = {
   formatRoleForSerialization: role => {
     const response = {_id: role._id};
 
-    response._source = _.assignIn({}, role);
+    response._source = _.assignIn({}, _.omit(role, ['_kuzzle_info']));
```

Comment:
`_.omit` already returns a copy of the provided object (without the properties to be omitted), so using `_.assignIn` on an empty object to create another copy is superfluous